plugins/actix-web/src/lib.rs

```diff
@@ -229,10 +231,16 @@ where
     /// recorded by the wrapper and serves them in the given path
     /// as a JSON.
     pub fn with_json_spec_at(mut self, path: &str) -> Self {
-        self.inner = self.inner.service(
-            actix_web::web::resource(path)
-                .route(actix_web::web::get().to(SpecHandler(self.spec.clone()))),
-        );
+        self.inner = self
```

Comment:
Let's rename the method to `with_swagger_spec_at`, as it feels more appropriate than having both HTML and JSON in a method that's named "json" - once that's done, we should also update the book.